tw-NEDa-frontend/src/app/components/location/location.component.html

```diff
@@ -0,0 +1,155 @@
+<div class="page-content">
+
+    <h2>Location</h2>
+    <div class="jumbotron">
+
+        <div class="row content">
+
+            <!-- 2 boxes where we filter the data (City Search/Magnitude search)-->
+            <div class="col-md-5 ng-scope" id="full">
+                <!-- DISTRICT SEARCH -->
+                <div class="panel panel-default">
+
+                    <!--City search -->
+                    <div class="panel-heading">
+                        <!--title form-->
+                        <h2 class="panel-title text-center">
+                            Search By District
+                            <!--insert icon-->
+                            <span class="glyphicon glyphicon-map-marker"></span>
+                        </h2>
+                    </div>
+                    <!--panel body-->
+                    <div class="panel-body">
+                        <form name="cityForm" novalidate>
+                            <div class="form-group">
+                                <!--selects names for the cities from location.component.ts -->
+                                <!-- (change)- sort of function which will keep up with every change-->
+                                <!-- I send the City name to trigger function -->
+                                <select (change)="triggerDistrict($event.target.value)">
+                                    <option
+                                            *ngFor="let city of cities"
+                                            [value]="city.value">
+                                        {{ city.viewValue}}
+                                    </option>
+                                </select>
+                                <!--each city has it's coordinates goToCity() moves google map to those coordinates-->
+                                <button (click)="goToDistrict()" id="city" md-button>Search District</button>
```

Comment:
I would suggest using a different material button so it can be easily seen (and known it's a button).

See [here](https://material.angular.io/components/component/button)